doc/api/buffer.md

```diff
@@ -100,7 +100,7 @@ to one of these new APIs.*
 * [`Buffer.from(buffer)`][] returns a new `Buffer` that *contains a copy* of the
   contents of the given `Buffer`.
 * [`Buffer.from(string[, encoding])`][`Buffer.from(string)`] returns a new
-  `Buffer` that *contains a copy* of the provided string.
+  `Buffer` that *contains a copy* of the provided string. (When `'hex'` encoding a string, using odd number of characters may result in an unpredictable behaviour).
```

Comment:
Given that this affects every Node.js API that takes an `encoding` argument (which is many), describing the behaviour of invalid input would be better placed at https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings

Also, the phrase added isn't so useful, I can't think of a predictable behaviour for providing a hex string that has half-bytes. Maybe describe what it actually does, ignore input from the first invalid or incomplete hex pair?

Reply:
Got you, on it.